deploy/008-deploy-vaults.ts

```diff
@@ -0,0 +1,64 @@
+import deployedContracts from "@venusprotocol/governance-contracts/deployments/deployments.json";
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts } = hre;
+  const { deploy } = deployments;
+  const { deployer } = await getNamedAccounts();
+
+  const xvsVaultDeployment = await deploy("XVSVault", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvsVaultAddress = xvsVaultDeployment.address;
+
+  const xvsVaultProxyDeployment = await deploy("XVSVaultProxy", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvsVaultProxyAddress = xvsVaultProxyDeployment.address;
+
+  await deploy("XVSStore", {
+    from: deployer,
+    args: [],
+    log: true,
+    autoMine: true,
+  });
+
+  const xvs = await ethers.getContract("XVS");
```

Comment:
We should fetch XVS address from deployments file because remote XVS token contract would be different from the current one we have on bsc

Reply:
If we have the contract deployed and exported under this name wouldn't hardhat pick the correct deployment/ address based on the network?

Reply:
I think we need to create instance by using address from token-bridge package

Reply:
I've added the token bridge package to the external deployments in hardhat. This should pull the XVS address correctly from the deployments based on the network